packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.cpp

```diff
@@ -0,0 +1,255 @@
+                if (!propertyValue.IsNull()) {
+                    expander.Width(propertyValue.AsDouble());
+                    this->Width(propertyValue.AsDouble());
+                }
+            }
+            else if (propertyName == "height") {
+                if (!propertyValue.IsNull()) {
+                    expander.Height(propertyValue.AsDouble());
+                    this->Height(propertyValue.AsDouble());
+                }
+            }
+            else if (propertyName == "contentHorizontalAlignment") {
+                if (!propertyValue.IsNull()) {
+                    auto alignment = propertyValue.AsString();
+                    if (alignment == "center") {
+                        expander.HorizontalContentAlignment(Windows::UI::Xaml::HorizontalAlignment::Center);
+                    }
+                    else if (alignment == "left") {
+                        expander.HorizontalContentAlignment(Windows::UI::Xaml::HorizontalAlignment::Left);
+                    }
+                    else if (alignment == "right") {
+                        expander.HorizontalContentAlignment(Windows::UI::Xaml::HorizontalAlignment::Right);
+                    }
+                    else if (alignment == "stretch") {
+                        expander.HorizontalContentAlignment(Windows::UI::Xaml::HorizontalAlignment::Stretch);
+                    }
+                }
+            }
+            else if (propertyName == "contentVerticalAlignment") {
+                if (!propertyValue.IsNull()) {
+                    auto alignment = propertyValue.AsString();
+                    if (alignment == "bottom") {
+                        expander.VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Bottom);
+                    }
+                    else if (alignment == "center") {
+                        expander.VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Center);
+                    }
+                    else if (alignment == "stretch") {
+                        expander.VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Stretch);
+                    }
+                    else if (alignment == "top") {
+                        expander.VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Top);
+                    }
+                }
+            }
+            else if (propertyName == "headerBackground") {
+                this->SetResourceColor(propertyValue, L"ExpanderHeaderBackground");
+            }
+            else if (propertyName == "headerForeground") {
+                this->SetResourceColor(propertyValue, L"ExpanderHeaderForeground");
+            }
+            else if (propertyName == "headerBorderThickness") {
+                if (!propertyValue.IsNull()) {
+                    auto resDict = expander.Resources();
+                    Windows::UI::Xaml::Thickness thickness = Windows::UI::Xaml::ThicknessHelper::FromUniformLength(propertyValue.AsDouble());
```

Comment:
nit can be auto

Reply:
also i think you're alising this namespace so winrt:: might work if i'm looking correctly